src/relay.js

```diff
@@ -20,41 +20,61 @@ import _ from 'lodash';
 import simplifyAST from './simplifyAST';
 
 class NodeTypeMapper {
-  constructor(sequelize) {
-    this.models = Object.keys(sequelize.models);
-    this.models.forEach(model => {
-      this[model] = null;
-    });
+  constructor() {
+    this.map = { };
   }
 
-  mapTypes(types) {
-    Object.keys(types).forEach(type => {
-      this[type] = types[type];
-    });
+  mapTypes( types ) {
+    this.map = {
+      ...this.map,
+      ...types,
+    };
+  }
+
+  item( type ) {
+    const nodeType = this.map[type];
+    return nodeType.type
+      ? nodeType
+      : { type: nodeType };
```

Comment:
Ok, I can do that, but my way is more efficient as it avoids iterating the keys of the types object.